templates/datagrid72-basic.json

```diff
@@ -5,19 +5,18 @@
         "annotations": {
             "iconClass": "icon-datagrid",
             "tags": "datagrid,jboss",
-            "version": "1.4.12",
```

Comment:
This is the templates version, but "1.4.12" refers to the application-templates repo (when they all lived together - the current version is "1.4.15"). However, once we separated ours from the original repo, we can version them as we see fit - in this case, I'm using the same version as the image, simplifying it. This was suggested by Rob - he said the version number and tag should be in sync, since the version number was added to make easier for a customer contacting support to know which tag is being used. It doesn't have any other effect other than labeling the templates.

Reply:
Ok makes sense, thanks for the explanation.